--- origsrc/avidemux_2.5.5/CMakeLists.txt	2011-11-28 23:39:51.500693500 -0600
+++ src/avidemux_2.5.5/CMakeLists.txt	2011-11-28 04:05:39.000000000 -0600
@@ -48,20 +48,15 @@ ENDIF (NOT CMAKE_BUILD_TYPE)
 ########################################
 INCLUDE(admDetermineSystem)
 
-IF (CYGWIN)
-	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mwin32")
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mwin32")
-ENDIF (CYGWIN)
-
 IF (ADM_CPU_ALTIVEC)
 	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 ENDIF (ADM_CPU_ALTIVEC)
 
-IF (UNIX AND NOT APPLE)
+IF (UNIX AND NOT APPLE AND NOT CYGWIN)
 	# jog shuttle is only available on Linux due to its interface
 	SET(USE_JOG 1)
-ENDIF (UNIX AND NOT APPLE)
+ENDIF (UNIX AND NOT APPLE AND NOT CYGWIN)
 
 IF (WIN32)
 	SET(BIN_DIR .)
--- origsrc/avidemux_2.5.5/addons/avsfilter/CMakeLists.txt	2010-09-09 09:36:36.000000000 -0500
+++ src/avidemux_2.5.5/addons/avsfilter/CMakeLists.txt	2011-11-28 23:34:23.255919000 -0600
@@ -1,4 +1,4 @@
-IF (NOT WIN32)
+IF (NOT WIN32 AND NOT CYGWIN)
     message(STATUS "avsfilter binary uncompressed")
     set(AVSFILTER_WINE_BINARY_ARCHIVE ${AVIDEMUX_SOURCE_DIR}/addons/avsfilter/avsload_pipe_binary.tar.bz2)
     set(addonesInstallDir ${CMAKE_INSTALL_PREFIX}/share/ADM_addons/avsfilter)
@@ -6,4 +6,4 @@ IF (NOT WIN32)
     find_package(Tar)
     execute_process(COMMAND ${TAR_EXECUTABLE} xvf ${AVSFILTER_WINE_BINARY_ARCHIVE} -C ${tempAvsFilterUnpackDir})
     install(FILES ${tempAvsFilterUnpackDir}/avsload.exe ${tempAvsFilterUnpackDir}/pipe_source.dll DESTINATION ${addonesInstallDir})
-endif (NOT WIN32)
+endif (NOT WIN32 AND NOT CYGWIN)
--- origsrc/avidemux_2.5.5/avidemux/ADM_libraries/ADM_nvwa/debug_new.cpp	2011-11-28 23:39:51.502693600 -0600
+++ src/avidemux_2.5.5/avidemux/ADM_libraries/ADM_nvwa/debug_new.cpp	2011-11-28 04:05:39.000000000 -0600
@@ -293,10 +293,9 @@ static bool print_position_from_addr(con
 #else
         const int  exeext_len = 0;
 #endif
-#if  !defined(__CYGWIN__) && defined(__unix__)
+#if   defined(__unix__)
         const char ignore_err[] = " 2>/dev/null";
-#elif defined(__CYGWIN__) || \
-        (defined(_WIN32) && defined(WINVER) && WINVER >= 0x0500)
+#elif (defined(_WIN32) && defined(WINVER) && WINVER >= 0x0500)
         const char ignore_err[] = " 2>nul";
 #else
         const char ignore_err[] = "";
--- origsrc/avidemux_2.5.5/cmake/admDetermineSystem.cmake	2011-11-28 23:39:51.506693900 -0600
+++ src/avidemux_2.5.5/cmake/admDetermineSystem.cmake	2011-11-28 04:05:39.000000000 -0600
@@ -69,12 +69,6 @@ IF (UNIX_SUPPORTED)
 
 		IF (CYGWIN_SUPPORTED)
 			SET(CYGWIN 1)
-
-			PERFORM_SYSTEM_TEST(os_win32_check.cpp "Win32" WIN32_SUPPORTED "-mwin32")
-
-			IF (WIN32_SUPPORTED)
-				SET(WIN32 1)
-			ENDIF (WIN32_SUPPORTED)
 		ENDIF (CYGWIN_SUPPORTED)
 	ENDIF (BSD_SUPPORTED)
 ELSE (UNIX_SUPPORTED)
--- origsrc/avidemux_2.5.5/cmake/adm_log.cmake	2011-11-28 23:39:51.504693700 -0600
+++ src/avidemux_2.5.5/cmake/adm_log.cmake	2011-11-28 04:05:39.000000000 -0600
@@ -70,7 +70,7 @@ MESSAGE( STATUS "CMAKE_SYSTEM_PROCESSOR:
 # is TRUE on all UNIX-like OS's, including Apple OS X and CygWin
 MESSAGE( STATUS "UNIX: " ${UNIX} )
 
-# is TRUE on Windows, including CygWin 
+# is TRUE on Windows
 MESSAGE( STATUS "WIN32: " ${WIN32} )
 
 # is TRUE on Apple OS X
--- origsrc/avidemux_2.5.5/plugins/ADM_videoFilters/CMakeLists.txt	2010-09-08 15:35:34.000000000 -0500
+++ src/avidemux_2.5.5/plugins/ADM_videoFilters/CMakeLists.txt	2011-11-28 23:24:35.772316800 -0600
@@ -1,8 +1,8 @@
 ADD_SUBDIRECTORY(AddBorders)
 
-IF (NOT WIN32 AND NOT APPLE)
+IF (NOT WIN32 AND NOT APPLE AND NOT CYGWIN)
 	ADD_SUBDIRECTORY(AvsFilter)
-ENDIF (NOT WIN32 AND NOT APPLE)
+ENDIF (NOT WIN32 AND NOT APPLE AND NOT CYGWIN)
 
 ADD_SUBDIRECTORY(BlendRemover)
 ADD_SUBDIRECTORY(Chroma)
@@ -44,9 +44,9 @@ ADD_SUBDIRECTORY(MSmooth)
 ADD_SUBDIRECTORY(Soften)
 ADD_SUBDIRECTORY(lavDeinterlace)
 
-IF (NOT WIN32 AND NOT APPLE)
+IF (NOT WIN32 AND NOT APPLE AND NOT CYGWIN)
   ADD_SUBDIRECTORY(Logo)
-ENDIF (NOT WIN32 AND NOT APPLE)
+ENDIF (NOT WIN32 AND NOT APPLE AND NOT CYGWIN)
 
 ADD_SUBDIRECTORY(ChromaShift)
 ADD_SUBDIRECTORY(CNR2)
--- origsrc/avidemux_2.5.5/plugins/CMakeLists.txt	2011-11-28 23:39:51.508694000 -0600
+++ src/avidemux_2.5.5/plugins/CMakeLists.txt	2011-11-28 04:05:39.000000000 -0600
@@ -88,11 +88,6 @@ ENDIF (NOT CMAKE_BUILD_TYPE)
 ########################################
 INCLUDE(admDetermineSystem)
 
-IF (CYGWIN)
-	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mwin32")
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mwin32")
-ENDIF (CYGWIN)
-
 IF (ADM_CPU_ALTIVEC)
 	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ADM_ALTIVEC_FLAGS}")
