--- origsrc/avidemux_2.5.1/CMakeLists.txt	2009-08-16 04:37:34.000000000 -0500
+++ src/avidemux_2.5.1/CMakeLists.txt	2009-11-10 22:31:13.075083800 -0600
@@ -48,20 +48,15 @@ ENDIF (NOT CMAKE_BUILD_TYPE)
 ########################################
 INCLUDE(admDetermineSystem)
 
-IF (CYGWIN)
-	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mwin32")
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mwin32")
-ENDIF (CYGWIN)
-
 IF (ADM_CPU_ALTIVEC)
 	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 ENDIF (ADM_CPU_ALTIVEC)
 
-IF (UNIX AND NOT APPLE)
+IF (UNIX AND NOT APPLE AND NOT CYGWIN)
 	# jog shuttle is only available on Linux due to its interface
 	SET(USE_JOG 1)
-ENDIF (UNIX AND NOT APPLE)
+ENDIF (UNIX AND NOT APPLE AND NOT CYGWIN)
 
 IF (WIN32)
 	SET(BIN_DIR .)
--- origsrc/avidemux_2.5.1/avidemux/ADM_libraries/ADM_nvwa/debug_new.cpp	2007-08-29 14:45:10.000000000 -0500
+++ src/avidemux_2.5.1/avidemux/ADM_libraries/ADM_nvwa/debug_new.cpp	2009-11-10 21:38:24.031325600 -0600
@@ -293,10 +293,9 @@ static bool print_position_from_addr(con
 #else
         const int  exeext_len = 0;
 #endif
-#if  !defined(__CYGWIN__) && defined(__unix__)
+#if   defined(__unix__)
         const char ignore_err[] = " 2>/dev/null";
-#elif defined(__CYGWIN__) || \
-        (defined(_WIN32) && defined(WINVER) && WINVER >= 0x0500)
+#elif (defined(_WIN32) && defined(WINVER) && WINVER >= 0x0500)
         const char ignore_err[] = " 2>nul";
 #else
         const char ignore_err[] = "";
--- origsrc/avidemux_2.5.1/cmake/adm_log.cmake	2008-03-12 14:10:21.000000000 -0500
+++ src/avidemux_2.5.1/cmake/adm_log.cmake	2009-11-10 20:24:51.558859300 -0600
@@ -70,7 +70,7 @@ MESSAGE( STATUS "CMAKE_SYSTEM_PROCESSOR:
 # is TRUE on all UNIX-like OS's, including Apple OS X and CygWin
 MESSAGE( STATUS "UNIX: " ${UNIX} )
 
-# is TRUE on Windows, including CygWin 
+# is TRUE on Windows
 MESSAGE( STATUS "WIN32: " ${WIN32} )
 
 # is TRUE on Apple OS X
--- origsrc/avidemux_2.5.1/cmake/admDetermineSystem.cmake	2008-03-21 19:10:07.000000000 -0500
+++ src/avidemux_2.5.1/cmake/admDetermineSystem.cmake	2009-11-10 21:39:03.141379600 -0600
@@ -69,12 +69,6 @@ IF (UNIX_SUPPORTED)
 
 		IF (CYGWIN_SUPPORTED)
 			SET(CYGWIN 1)
-
-			PERFORM_SYSTEM_TEST(os_win32_check.cpp "Win32" WIN32_SUPPORTED "-mwin32")
-
-			IF (WIN32_SUPPORTED)
-				SET(WIN32 1)
-			ENDIF (WIN32_SUPPORTED)
 		ENDIF (CYGWIN_SUPPORTED)
 	ENDIF (BSD_SUPPORTED)
 ELSE (UNIX_SUPPORTED)
--- origsrc/avidemux_2.5.1/plugins/CMakeLists.txt	2009-02-21 11:26:30.000000000 -0600
+++ src/avidemux_2.5.1/plugins/CMakeLists.txt	2009-11-10 21:39:03.151379600 -0600
@@ -78,11 +78,6 @@ ENDIF (NOT CMAKE_BUILD_TYPE)
 ########################################
 INCLUDE(admDetermineSystem)
 
-IF (CYGWIN)
-	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mwin32")
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mwin32")
-ENDIF (CYGWIN)
-
 IF (ADM_CPU_ALTIVEC)
 	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ADM_ALTIVEC_FLAGS}")

